### PR TITLE
update model

### PR DESCRIPTION
**Pull request by SoniaChoo**
update all needed models
the database schema updated, we will update models to make consistent changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 a microservice online store to buy and sell
 
 * The poject's struct is showed as follow:
-![Architect](WebServiceDemo_V1.png)
\ No newline at end of file
+![Architect](WebServiceDemo_V1.png)
```
Comment (ziyitony) [inline README.md:5]: why change this line?

File: model/dish.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+type Dish struct {
+	DishId int
+	RestId int
+	Price float64
+	DishName string
+	Stock int
+	Sales int
+	CreatTime time.Time
+}
```
Comment (ziyitony) [inline model/dish.go]: please add one more line at EOF(end of file)

File: model/user.go [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-	NewEmail string
-	NewPasswd string
-}
+type User struct {
+	UserId int
+	Phone string
+	Nickname string
+	Password string
+	CreatTime time.Time
+}
```
Comment (ziyitony) [inline model/user.go]: please add one more line at EOF

**Review by ziyitony — CHANGES_REQUESTED** [review]
please address comments and **format all files**.

File: README.md (hunk 1 above)
Comment (SoniaChoo) [reply README.md:5]: forgot to add newline at the end of file last time.

File: model/dish.go (hunk 2 above)
Comment (SoniaChoo) [reply model/dish.go]: thanks, updated.

File: model/user.go (hunk 3 above)
Comment (SoniaChoo) [reply model/user.go]: thanks, updated

**Comment by SoniaChoo** [comment]
> please address comments and **format all files**.

thanks, updated and formatted.

**Review by ziyitony — APPROVED** [review]
nice job

**Pull request merged**